salt/states/blockdev.py

```diff
@@ -126,6 +126,13 @@ def formatted(name, fs_type='ext4', **kwargs):
 
     fs_type
         The filesystem it should be formatted as
+
+    force
+        Force mke2fs to create a filesystem, even if the specified device is
+        not a partition on a block special device. This option is only enabled
+        for ext and xfs filesystems
+
+        This option is dangerous, use it with caution.
```

Comment:
@abednarik, `.. versionadded:: Carbon`.  Make sure this has the same indent as the text describing `force`.